### PR TITLE
Task/um 697 update memory resource docs

### PR DESCRIPTION
**Pull request by kab163**

### CODE REVIEW

**Comment by kab163** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Changelog
-
+
```
Comment (davidbeckingsale) [inline CHANGELOG.md:2]: ?

**Review by davidbeckingsale — CHANGES_REQUESTED** [review]
Can you add a real CHANGELOG update?

File: CHANGELOG.md (hunk 1 above)
Comment (kab163) [reply CHANGELOG.md:2]: Lol sorry, this is me being weird and getting frustrated with git - I'll take that off

**Comment by kab163** [comment]
LGTM

**Review by davidbeckingsale — APPROVED** [review]

**Pull request merged**